tests/profiler/tensorflow2/test_native_tf2_profiler.py

```diff
@@ -108,6 +111,33 @@ def _verify_tensor_names(out_dir):
     assert trial.tensor_names(collection=CollectionKeys.OUTPUTS) == ["labels", "logits"]
 
 
+def _verify_detailed_profiling(out_dir):
+    """
+    This verifies the number of events when detailed profiling is enabled.
+    """
+    t_events = TensorboardProfilerEvents()
+
+    # get tensorboard timeline files
+    files = []
+
+    for path in Path(os.path.join(out_dir, "framework")).rglob(f"*{TENSORBOARDTIMELINE_SUFFIX}"):
```

Comment:
Try to make this a helper function or a `fixture` if possible

Reply:
Made it a helper function.